git_big/main.py

```diff
@@ -861,22 +877,33 @@ def help(ctx, topic, **kw):
 
 @cli.command('version')
 def cmd_version():
-    '''Print version and exit.'''
+    """Print version and exit."""
     click.echo(__version__)
 
 
 @cli.command('init')
 def cmd_init():
-    '''Initialize a repository.'''
+    """Initialize a repository."""
     App().cmd_init()
 
 
+@cli.command('set-depot')
+@click.option('--url', prompt='Depot URL', help='The URL of the git-big depot')
+@click.option('--secret', prompt=True, help='The git-big depot secret')
```

Comment:
If these are all required, then they should be arguments, not options.

Reply:
http://click.pocoo.org/6/parameters/

I think options are a better fit for click's model of the world.  Not that we have to stick with that model, but it's a bit more work to go around it.

Reply:
Fair enough, I suppose the prompting only works with options.